repose-aggregator/src/docs/asciidoc/release-notes.adoc

```diff
@@ -1,5 +1,9 @@
 = Release Notes
 
+== 8.7.0.2 (2017-10-04)
+
+* https://repose.atlassian.net/browse/REP-6162[REP-6162] - Updated the Keystone v2 get IDP call to support the field name change from `approvedDomains` to `approvedDomainIds`.
+
```

Comment:
Now with release notes!

I went ahead and stuck 8.7.0.2 in here since I expect that is the version we are targeting? I also set the date.